### ISSUE TITLE
remove woven usage from twisted.web.distrib

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3559]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#3559|
|Type|task|
|Created|2008-12-01 17:35:10Z|
|Branch|https://github.com/twisted/twisted/tree/non-woven-distrib-3559|

twisted.web.woven is deprecated, nothing in twisted should use it anymore.

<details><summary>Searchable metadata</summary>

```
trac-id__3559 3559
type__task task
reporter__exarkun exarkun
priority__highest highest
milestone__Twisted_9_0 Twisted-9.0
branch__branches_non_woven_distrib_3559 branches/non-woven-distrib-3559
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__web web
keywords__ 
time__1228152910000000 1228152910000000
changetime__1229357730000000 1229357730000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [25582]) Branching to 'non-woven-distrib-3559'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_5"></a>|@therve set owner to @exarkun|
|-|-|

Only one nit:
`+        return self.template % {'users' :listing.toxml()}`

Please move the space before ':'.

+1.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun commented|
|-|-|

(In [25668]) fix dict literal whitespace

refs [#3559](https://github.com/twisted/twisted/issues/3559)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun set status to `closed`|
|-|-|

(In [25670]) Merge non-woven-distrib-3559

Author: exarkun
Reviewer: therve
Fixes: [#3559](https://github.com/twisted/twisted/issues/3559)

Rewrite the distributed user directory resource to do its own HTML
generation instead of using the long-deprecated `twisted.web.woven`
HTML generation features.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
